### PR TITLE
Skeletonize the project

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request skeletonizes this project on [cisagov/skeleton-docker](https://github.com/cisagov/skeleton-docker).

## 💭 Motivation and context ##

This first step in modernizing any of our projects is to ensure they are on an appropriate skeleton project so they are integrated with [Lineage](https://github.com/cisagov/action-lineage).

### ❗ Note ###

The base image chosen for the `Dockerfile` mirrors the version of Python used in the `python:slim-stretch` image when [`dhsncats/client-cert-update:0.0.2`](https://hub.docker.com/layers/client-cert-update/dhsncats/client-cert-update/0.0.2/images/sha256-a59b3e021ae83f25a87adc87e18a42741ca1247d0ff84c3a572689d1af1b1eac?context=explore) was built.

## 🧪 Testing ##

Automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Bump version.

## ✅ Post-merge checklist ##

- [ ] Add a tag or create a release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,243 @@
...
+    docker compose up --detach
+    ```
+
+## Using secrets with your container ##
+
+This container also supports passing sensitive values via [Docker
+secrets](https://docs.docker.com/engine/swarm/secrets/).  Passing sensitive
+values like your credentials can be more secure using secrets than using
+environment variables.  See the
+[secrets](#secrets) section below for a table of all supported secret files.
```
Comment (dav3r) [inline README.md:56]: Odd line breaking here.  Why not this?
```suggestion
environment variables.  See the [secrets](#secrets) section below for a table
of all supported secret files.
```

**Review by dav3r — APPROVED** [review]
Looks great!  Thanks for skeletonizing yet another old, dusty repo! ☠️

File: README.md (hunk 1 above)
Comment (mcdonnnj) [reply README.md:56]: I'm just usin' [what I was given](https://github.com/cisagov/skeleton-docker/blob/9cea64da4a1379cdde3c509a09fc548f0da8b034/README.md?plain=1#L63-L65).

Comment (dav3r) [reply README.md:56]: What the hell was @jsf9k thinking? 😝

Comment (jsf9k) [reply README.md:56]: Who knows...

**Review by jsf9k — APPROVED** [review]
This all looks good to me.

File: README.md (hunk 1 above)
Comment (dav3r) [reply README.md:56]: Resolving this conversation.  @mcdonnnj - feel free to accept my suggestion above or not.

Comment (mcdonnnj) [reply README.md:56]: @dav3r I'll take it up to [cisagov/skeleton-docker](https://github.com/cisagov/skeleton-docker).

**Pull request merged**